testgen/src/main/groovy/GenerateApiDocsTest.groovy

```diff
@@ -0,0 +1,54 @@
+package com.vivy.springdoc
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.TaskAction
+
+class GenerateApiDocsTest extends DefaultTask {
+
+    @Input
+    String openApiDocsTestOut;
+    @Input
+    String testPackage;
+    @Input
+    @Optional
+    String extensionClassImport = null;
```

Comment:
instead better to just specify a fqdn of the `parentClass`
```suggestion
    String parentClass = null;
```

and it couldn't be optional, as otherwise how would you inject anything. Also with the webflux comes `WebTestClient`

Reply:
> instead better to just specify a fqdn of the `parentClass`

could you please bring a sample?

> and it couldn't be optional, as otherwise how would you inject anything. 

What if service does not use any basic class for testing (Means no `Abstract integration class`)

> Also with the webflux comes `WebTestClient`

Does this mean I have to add any import?
How the naïve implementation works then?

Reply:
If service doesn't use any kind of extension - it couldn't autowire anything. I thing rest template comes from spring-test starter, if that's the case - all fine, lets skip it for now.